telemetry/csharp/ToolkitTelemetryGenerator/DefinitionsBuilder.cs

```diff
@@ -0,0 +1,757 @@
+﻿using System;
+using System.CodeDom;
```

Comment:
Is this like a built-in code generator? That's pretty cool.

Reply:
Yup, its pretty slick! https://docs.microsoft.com/en-us/dotnet/framework/reflection-and-codedom/using-the-codedom

It sounds like .NET 5 will feature a more comprehensive source code generator, but this suits our immediate needs.